open-redirect.yaml

```diff
@@ -0,0 +1,31 @@
+rules:
+- id: open-redirect
+  mode: taint
+  pattern-sources:
+  - patterns:
+    - pattern: $REDIRECTURL
+  pattern-sinks:
+  - patterns:
+    - pattern-inside: 'Redirect($REDIRECTURL)
+
+        '
+  pattern-sanitizers:
+  - pattern: "if (IsLocalUrl($REDIRECTURL)) \n{ \n    ... Redirect($REDIRECTURL);\
```

Comment:
Sanitizers usually are functions that help with escaping or input filtering (for instance `htmlspecialchars` in PHP). If you're trying to capture cases where `IsLocalUrl` is called in an if statement, it might be better to use `pattern-not-inside` in the `sinks` category.